### ISSUE TITLE
[All][Accessibility] Voice Over issue in Input Field Label

### ISSUE THREAD

**Issue by sowrabh-msft** [microsoft/AdaptiveCards#4898]
# All platforms
- [x]  Android (#4930)
- [x] iOS (#4951)

# Host

Microsoft Teams

# Version of SDK

SDK version 2.1.0 (ACv1.3)

# Details

When the user goes into the card and selects/shifts focus to an input field, the field label is not called out which results in users being unable to identify what they are supposed to fill in the particular field. Only an issue on the mobile platforms.

Fix: Call out input label as well while emphasis is on field. 
Link to [video](https://microsoft-my.sharepoint.com/personal/shisaha_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fshisaha%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3%2FAccessibility%2Finput%5Ffield%5Faccessibility%2Emp4&parent=%2Fpersonal%2Fshisaha%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3%2FAccessibility&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOnY6L3Avc2hpc2FoYS9FVk9CN2xqOUh3OUhzendXOE8yUF9IOEJsV003Qk1JbFFoUUhQSGtMZzk2RG9BP3J0aW1lPU1yN1hWZkoxMkVn)

**Comment by shalinijoshi19** [comment]
@sowrabh-msft is there guidance around localizing the voice over text here at all to reconcile with the user locale settings? 
@paulcam206 / @almedina-ms  is this something that we are aware of with voiceover/narrator?

**Comment by shiladityasaha** [comment]
Adding video links for easy access: 
[Item #1: Video](https://microsoft-my.sharepoint.com/personal/shisaha_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fshisaha%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3%2FAccessibility%2Finput%5Ffield%5Faccessibility%2Emp4&parent=%2Fpersonal%2Fshisaha%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3%2FAccessibility&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOnY6L3Avc2hpc2FoYS9FVk9CN2xqOUh3OUhzendXOE8yUF9IOEJsV003Qk1JbFFoUUhQSGtMZzk2RG9BP3J0aW1lPTk1QVdXdzlzMkVn)
[Item #2: Video](https://microsoft-my.sharepoint.com/personal/dipja_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fdipja%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3%2Ferror%5Faccessibility%2Emp4&parent=%2Fpersonal%2Fdipja%5Fmicrosoft%5Fcom%2FDocuments%2FACv1%2E3&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOnY6L3AvZGlwamEvRVE1Uzc0aUNwTmxObXBFRURHV1RqcTRCSUpYREI1NUN6Tnk5dnUyQUdEWEZTZz9ydGltZT1MSHNub0E5czJFZw)

**Comment by jwoo-msft** [comment]
possible 20.10 if it's P0

**Comment by shiladityasaha** [comment]
@shalinijoshi19 this is a ring blocker on all platforms for us. Hopefully this can be picked up for desktop too?

**Comment by dclaux** [comment]
I am not able to repro this in Edge or Chrome on Windows with Narrator. All the appropriate From the first video (I don't have access to the other two) it looks like the problem happens in WebChat on iOS, is that correct?

I am not able to repro this in Edge or Chrome on Windows with Narrator. All the appropriate aria properties are set, and narration is as expected.

I suspect this is a case of WebChat not being updated to use the latest AC SDK in which a number of accessibility bugs were fixed.

**Comment by golddove** [comment]
@shiladityasaha we have split the bug you've filed, as it had two separate issues. The second part has been moved to #4912 (and related #4913 ). So, I think you are referring to that second part for desktop?

This issue now only tracks the first part, which should only be on mobile. Please confirm if that's correct.

**Comment by golddove** [comment]
For Android, likely fixed by #4930. Need to test.

**Comment by shalinijoshi19** [comment]
- Pending validation for Android - @golddove ; 
- iOS in code review- @jwoo-msft

**Comment by jwoo-msft** [comment]
@shalinijoshi19 This issue will be taken together with #4912

**Comment by golddove** [comment]
Validated on Android. Works on `v1.3/Tests/InputValidationTestCard.json` as expected (reads the associated label for each EditText).
